methods.py

```diff
@@ -989,7 +989,7 @@ def progress_finish(target, source, env):
             pass
 
     try:
-        with open(node_count_fname) as f:
+        with open(node_count_fname, "r", encoding="utf-8") as f:
```

Comment:
It's written to using `utf-8` encoding: https://github.com/godotengine/godot/blob/38716b94dbcc9dbfb60a49399c5118f01fc79c45/methods.py#L985-L986